src/view/src/main.rs

```diff
@@ -35,6 +37,22 @@ fn parse(args: Args) -> View {
             let path = PathBuf::from_str(&arg).unwrap();
             view.path = path;
         }
+
+        if arg == &r#"-o"# {
```

Comment:
The raw string here is not actually required, this syntax is better used when you want to have multiline strings in your source.

Lets do:

```suggestion
        if arg == "-o" {
```

Instead